1es-azure-pipeline.yml

```diff
@@ -0,0 +1,84 @@
+trigger:
+  batch: true
+  branches:
+    include:
+    - main
```

Comment:
nit: Generally - feature/* is also good addition here

Reply:
I think this is a good idea, but we've never used a feature branch before in this repo, and Yanni is the first one to use it on the SDK repo. This is one interesting way to try to prevent a public PR that fails private tests from being merged, but TBH I don't think I would remember to check it. For now, I will leave this out, but if you have other reasons for doing this then let me know!